app/assets/stylesheets/content/_tables.sass

```diff
@@ -54,6 +52,8 @@ table
         text-align: right
         a
           padding-right: 0.6em
+      .form--text-field, .form--select
```

Comment:
I think you also want to apply this to text areas:
![lower margin wrong](https://cloud.githubusercontent.com/assets/453584/6735309/a7b23b62-ce5d-11e4-98bc-8b9bb3eb7856.PNG)

Inspecting the elements you can see that a text area still has 16px margin at the bottom, I guess this is the one you are explicitly removing here for other elements.

*Note: If you wonder how, just add a custom field of type `Long Text`*